src/models/async/TranslationMode.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-shadow */
+/** Supported model types for translation. */
+export enum TranslationMode {
```

Comment:
why do we emphasize that for Summarization we use a Model, but for the translation we have a Mode.

Reply:
Consistency with the backend inconsistencies :-)))
@alexsku okay to rename to TranslationModel ?

Reply:
btw it looks like the backend has `Fast` mode also

Reply:
We're not supporting Fast at this point.
I checked with @alexsku and he suggested not removing it from the backend for now.

Reply:
while on a backend, we have `TranslationMode`, I see that all consumer-facing names are `model`.
how about keeping it as `Model` for both?

also, why is it a Model for Translation, but NLP Model for Summarization? why is it not a `SummarizationModel` instead of `NlpModel`?

Reply:
Historically this is how it was defined for Nexus.
Do you think it is worth renaming it at this point?

Reply:
Actually SummarizationModel is already in use...

Reply:
I think it does not matter how do we call it internally. from my understanding this is the first time we are exposing such a type. and it is always easier to have a tech debt to rename our internal code rather than making changes to the public API - users will hate us.

originally, it was `NlpModel` - some abstract model type offered by all NLP-related features (summarization, translation).
now, we want to make it more specific. so we have `TranslationModel` for the translation, which is pretty much specific. but `NlpModel` for Summarization. what if we have a topic extraction or any other API that can use different models? how would we call it?

Reply:
Don't get me wrong, I agree :-)
We can easily rename it in NodeJS SDK, it's just a bit more tricky in the backend.